packages/e2e-tests/src/tests/WrappedKeys.spec.ts

```diff
@@ -0,0 +1,1211 @@
+        address:
+          '4fd3d6ae41190cdd33a07bc5feb4a51b0c882474e6b51eb37cf799d6668eb44b',
+        algo: 'ed25519',
+      },
+    };
+
+    try {
+      const privateKey = randomSolanaPrivateKey();
+
+      await importPrivateKey({
+        pkpSessionSigs,
+        privateKey,
+        litNodeClient: devEnv.litNodeClient!,
+        publicKey: '0xdeadbeef',
+        keyType: 'K256',
+        memo: 'Test Key',
+      });
+    } catch (e: any) {
+      if (e.message.includes('Invalid sessionSig: Expired')) {
+        console.log('✅ THIS IS EXPECTED: ', e);
+      } else {
+        throw e;
+      }
+    }
+  });
+
+  it('Fail EOA Session Sig', async () => {
+    const alice = await devEnv.createRandomPerson();
+
+    const eoaSessionSigs = await getEoaSessionSigs(devEnv, alice);
+
+    console.log(eoaSessionSigs);
+
+    const privateKey = randomSolanaPrivateKey();
+
+    try {
+      await importPrivateKey({
+        pkpSessionSigs: eoaSessionSigs!,
+        privateKey,
+        litNodeClient: devEnv.litNodeClient!,
+        publicKey: '0xdeadbeef',
+        keyType: 'K256',
+        memo: 'Test Key',
+      });
+    } catch (e: any) {
+      if (e.message.includes('SessionSig is not from a PKP')) {
+        console.log('✅ THIS IS EXPECTED: ', e);
+      } else {
+        throw e;
+      }
+    }
+  });
+
+  it('Fail Eth Sign Tx Missing Params', async () => {
+    const alice = await devEnv.createRandomPerson();
+
+    const pkpSessionSigs = await getPkpSessionSigs(
+      devEnv,
+      alice,
+      undefined,
+      new Date(Date.now() + 1000 * 60 * 10).toISOString()
+    ); // 10 mins expiry
+
+    const privateKey = ethers.Wallet.createRandom().privateKey;
+
+    const { pkpAddress, id } = await importPrivateKey({
+      pkpSessionSigs: pkpSessionSigs!,
+      privateKey,
+      litNodeClient: devEnv.litNodeClient!,
+      publicKey: '0xdeadbeef',
+      keyType: 'K256',
+      memo: 'Test Key',
+    });
+
+    const alicePkpAddress = alice.authMethodOwnedPkp?.ethAddress;
+    expect(pkpAddress).toEqual(alicePkpAddress);
+
+    const pkpSessionSigsSigning = await getPkpSessionSigs(
+      devEnv,
+      alice,
+      undefined,
+      new Date(Date.now() + 1000 * 60 * 10).toISOString()
+    ); // 10 mins expiry
+
+    try {
+      await signTransactionWithEncryptedKey({
+        pkpSessionSigs: pkpSessionSigsSigning!,
+        network: 'evm',
+        unsignedTransaction: {
+          ...getChainForNetwork(devEnv.litNodeClient?.config?.litNetwork!),
+          // @ts-expect-error This test is intentionally using the type incorrectly.
+          serializedTransaction: 'random-value',
+        },
+        broadcast: false,
+        litNodeClient: devEnv.litNodeClient!,
+      });
+    } catch (e: any) {
+      if (
```

Comment:
We can use expect.toThrow() here instead, and :hocho: the extra `console.log`
See also https://jestjs.io/docs/expect#rejects :)